_pages/community.html

```diff
@@ -1,187 +1,285 @@
+          {% include icons/manage-search.svg %}
+          shards.info
+        </a>
+        <p >Lists shard repositories on GitHub and GitLab with dependencies and releases.</p>
+      </div>
+
+      <div class="link-item" id="crystalshards.org">
+        <a class="link-item__main" href="https://crystalshards.org">
+          {% include icons/manage-search.svg %}
+          crystalshards.org
+        </a>
+        <p>Lists shard repositories on GitHub with dependencies.</p>
+      </div>
+
+      <div class="link-item" id="awesome">
+        <a class="link-item__main" href="https://github.com/veelenga/awesome-crystal">
+          {% include icons/manage-search.svg %}
+          awesome-crystal
+        </a>
+        <p>A curated Awesome List of Crystal shards, tools and tutorials.</p>
+      </div>
+    </div>
+
+
+    <div class="link-items">
+      <h3>Tooling</h3>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://github.com/veelenga/awesome-crystal#editor-plugins">
+          {% include icons/extensions.svg %}
+          Editor Plugins
+        </a>
+        <p>Compilation of plugins for source editors and IDEs.</p>
+      </div>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://crystal-ameba.github.io/">
+          {% include icons/ameba.svg %}
+          Ameba
+        </a>
+        <p>Code style linter for Crystal</p>
+      </div>
+    </div>
+  </div>
+
+</section>
+
+<h2>Crystal Sphere</h2>
+
+<section>
+
+  <div class="topics">
+    <div class="link-items">
+      <h3>Podcasts & Videos</h3>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://podcast.chicagocrystal.org/">
+          {% include icons/podcasts.svg %}
+          Chicago Crystal Podcast
+        </a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://www.youtube.com/channel/UCI1RvHPG6S9mw4eRoJfH2kA">
+          {% include icons/subscriptions.svg %}
+          Chicago Crystal Videos</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://www.youtube.com/playlist?list=PLe1-A91ZPTpAIO9YRZbAlPT3l1S5pMPrX">
+          {% include icons/video-library.svg %}
+          Crystal 1.0 Conference Playlist
+        </a>
+      </div>
+    </div>
+    <div class="link-items">
+      <h3>Blogs</h3>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://dev.to/t/crystal">
+          {% include icons/dev.to.svg %}
+          Crystal on dev.to</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://medium.com/tag/crystal-lang">
+          {% include icons/medium.svg %}
+          Crystal on Medium</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://lbarasti.com/tags/crystal/">
+          {% include icons/newsmode.svg %}
+          Lorenzo Barasti</a>
+      </div>
```

Comment:
I think this belongs to the External resources page in the documentation, but the call is tight.

Reply:
In any case, the name isn't very telling :-D

Reply:
Not really, this is a personal blog post. The others are for anybody that wants to `tag:crystal`

Reply:
Perhaps I'm just missing what's the line to put things here

Reply:
Sure, dev.to and Medium are aggregates. But in the end they're just a collection of personal blogs. The only difference is that they're accessed via a shared provider vs. others are independent.

Reply:
OK, but I would at least link personal blog of people that regularly post about Crystal. Lorenzo might still make the cut (although the last one is from 3yrs ago...), and Manas definitively doesn't fit.

Reply:
I'd say to proceed conservatively, and then we can add as we see fit

Reply:
IMO regular posting should not be a necessary critererion. I'm more interested in high quality content that is (still) relevant. And I think both Lorenzo as well as Manas match that. But we're really diverting from the main purpose of implementing the design.
We can simply drop those two links, or the entire blogs section and then come back with a separate discussion about the contents.

Reply:
sounds good, let's leave it for now